src/CoreBundle/private/modules/MTHeader/MTHeader.class.php

```diff
@@ -764,9 +763,8 @@ public function GetHtmlHeadIncludes()
             $aIncludes[] = '<script src="'.TGlobal::GetStaticURLToWebLib('/javascript/cms.js').'" type="text/javascript"></script>';
         }
         $aIncludes[] = '<link href="'.TGlobal::GetStaticURLToWebLib('/javascript/jquery/jQueryUI/themes/cupertino/cupertino.css').'" media="screen" rel="stylesheet" type="text/css" />';
-        $aIncludes[] = '<link href="/chameleon/blackbox/bootstrap/css/bootstrap.min.css" media="screen" rel="stylesheet" type="text/css" />';
-        $aIncludes[] = '<link href="/chameleon/blackbox/bootstrap/css/bootstrap-theme.min.css" media="screen" rel="stylesheet" type="text/css" />';
-        $aIncludes[] = '<link href="'.TGlobal::GetPathTheme().'/css/bootstrap.css" media="screen" rel="stylesheet" type="text/css" />';
+        $aIncludes[] = '<link href="/chameleon/blackbox/bootstrap/css/bootstrap.min.css?v4.1" media="screen" rel="stylesheet" type="text/css" />';
```

Comment:
I suppose the "v4.1" parameter is for cache busting - wouldn't it make sense to specify the version down to patch level then?

Reply:
You are right. Fixed.